<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1330</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tbw_data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1001</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_method">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>540</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_setFile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>540</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Chsose file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>700</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stat">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>640</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Statistic</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_metric">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>590</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_parametr1">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>540</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_parametr2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>590</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_setNeuralnet">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>590</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Chsose neuralnet</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbw_net_tree">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>10</y>
      <width>281</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_addlayout">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>540</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавть слой</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dellayout">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>590</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Убрать слой</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_id">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>590</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spb_neirons">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>540</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="br_status"/>
 </widget>
 <resources/>
 <connections/>
</ui>
